packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -34,14 +34,18 @@
       @mixin set-font-size var(--atomic-text-sm);
 
       margin-bottom: 1rem;
-      height: 2rem;
+      --line-height: 2rem;
+      line-height: var(--line-height);
+      max-height: calc(2 * var(--line-height) + 0.5rem);
```

Comment:
Could we use a mixin to reduce the duplication?

Reply:
Take a look at `@mixin set-font-size` it does already set the line height, we would need to add max-height to it and ti should be good

Reply:
Instead, I'll move `line-height` and `max-height` into the result-with-sections file.